### DISCUSSION TITLE
VS code Wallbash doesn't works properly!

### DISCUSSION THREAD

**Discussion by KitsuneKode** [prasanthrangan/hyprdots#1754] [Q&A]
It may be the issue only with my device but I installed the VS code wallbash from the vsix file too! Every time the theme closes, the vs code needs to be restarted to get the theme changed! Further, all the syntax highlighting is weird, it sometimes behaves like using a notepad! I want to know whether there is any fix to it or if it's still in production. If it is possible to add some great syntax highlights, it helps differentiate things in a big codebase with eyes!

Moreover the wallbashcode.sh throws this error
 `jq: parse error: Invalid numeric literal at line 22, column 5
jq: parse error: Invalid numeric literal at line 22, column 5
` 

No idea about jq and the json file it's reading!

**Comment by kRHYME7** [comment]
It's expected to restart/reload the whole vscode window to make the themes effective. 

About the colors, the template can be modified to  have a better contrast. 

the wallbashcode errors you got is maybe due to your settings.json. `jq` is used to parse json files to read your current theme extension.
